autoware_sensing_msgs/msg/RadarObject.msg

```diff
@@ -0,0 +1,35 @@
+float32 INVALID_COV_VALUE = 100000000.0
```

Comment:
```suggestion
float32 INVALID_COV_VALUE = -1.0f;
```
https://en.wikipedia.org/wiki/Covariance_matrix
> Any [covariance](https://en.wikipedia.org/wiki/Covariance) matrix is [symmetric](https://en.wikipedia.org/wiki/Symmetric_matrix) and [positive semi-definite](https://en.wikipedia.org/wiki/Positive_semi-definite_matrix) and its main diagonal contains [variances](https://en.wikipedia.org/wiki/Variance) (i.e., the covariance of each element with itself).

And since the message contains the diagonal of this matrix in the arrays below, we could make the invalid value `-1.0f`.